ArduCopter/GCS_Mavlink.cpp

```diff
@@ -585,6 +542,7 @@ bool GCS_MAVLINK_Copter::try_send_message(enum ap_message id)
     case MSG_FENCE_STATUS:
     case MSG_WIND:
     case MSG_POSITION_TARGET_GLOBAL_INT:
+    case MSG_SERVO_OUT:
```

Comment:
does the removal of Copter::send_servo_out() mean that HIL is officaily dead now, or is it some place else?